src/Cache/RedisSentinelStore.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace TusPhp\Cache;
+
+use Carbon\Carbon;
+use TusPhp\Config;
+use Predis\Client as RedisClient;
+
+class RedisSentinelStore extends AbstractCache
+{
+    /** @var RedisClient */
+    protected $redis;
+
+    /**
+     * RedisStore constructor.
+     *
+     * @param array $options
+     */
+    public function __construct(array $params = [], array $options = [])
+    {
+        $options = empty($options) ? Config::get('sentinel.options') : $options;
+        $params = empty($params) ? Config::get('sentinel.params') : $params;
+
+        $this->redis = new RedisClient($params, $options);
```

Comment:
If we only need to pass missing option in RedisClient, then we can just update RedisStore to accept second parameter instead of writing a new implementation. This should not break backwards compatibility.

```sh
public function __construct(array $options = [], array $params = [])
```

instead of https://github.com/ankitpokhrel/tus-php/blob/20671422b433aa45196f89906acf7c4328043ab8/src/Cache/RedisStore.php#L19